FirstApp/View/WebViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  WebViewController.swift
+//  FirstApp
+//
+//  Created by Akua Afrane-Okese on 2020/03/03.
+//  Copyright © 2020 Akua Afrane-Okese. All rights reserved.
+//
+
+import UIKit
+import WebKit
+
+class WebViewController: UIViewController, WKNavigationDelegate {
+    @IBOutlet weak var webView: WKWebView!
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    var website = ""
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        super.viewDidLoad()
+        let web = website
+        let url = URL(string: web)!
+        webView = WKWebView(frame: self.view.frame)
+        webView.isUserInteractionEnabled = true
+        webView.navigationDelegate = self
+        self.view.addSubview(self.webView)
+        webView.load(URLRequest(url: url))
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lines should not have trailing whitespace.](https://app.codacy.com/manual/AkuaQ/MyFirstIoSApp/pullRequest?prid=5109779)